### ISSUE TITLE
The AUT shakes in Canary

### ISSUE THREAD

**Issue by jennifer-shehane** [cypress-io/cypress#1620]
### Current behavior: 

Application under test is :: shaking :: in Canary 68.0.3400.0. It looks like the scrollbar is duplicated? Taking up twice the space? Something is making the content displace, which I imagine is some bug in Canary - here to document and track mostly.

![ugdfowkupg](https://user-images.githubusercontent.com/1271364/39024858-8391da26-4411-11e8-955e-eea7bf1218f7.gif)

### Steps to reproduce:

I just ran the kitchen-sink

**Comment by egucciar** [comment]
Awesome that you experienced this, its happening to me and driving me crazy!!!!!!!

**Comment by dannynicolas** [comment]
Yep. Upgraded to Chrome 66 and this started happening to me too. 

@egucciar thanks for the suggestion of running in electron mode, that works for me.

**Comment by jennifer-shehane** [comment]
So, after taking a nice stroll through Chrome's bugs, I believe this may have to do with [this open bug](https://bugs.chromium.org/p/chromium/issues/detail?id=833167&q=scrollbar&colspec=ID%20Pri%20M%20Stars%20ReleaseBlock%20Component%20Status%20Owner%20Summary%20OS%20Modified), where the content inside an iframe, when absolute positioned is displaced by 17px (the exact amount of the scrollbar width). Haven't confirmed this completely yet or if Cypress could provide a workaround.

**Comment by jennifer-shehane** [comment]
I'm back with a fix! If you want the AUT to stop shaking (until Chrome issues a fix / Cypress disables this behavior by default) - you can pass this flag to Chrome: `--disable-blink-features=RootLayerScrolling`

**Example Workaround Code:**
```js
module.exports = (on) => {
  on('before:browser:launch', (browser = {}, args) => {
    if (browser.name === 'chrome') {
      args.push('--disable-blink-features=RootLayerScrolling')
      return args
    }
  })
}
```

**Comment by jennifer-shehane** [comment]
I meant to post the code for this from Cypress. This flag can be easily added here in a PR: https://github.com/cypress-io/cypress/blob/develop/packages/server/lib/browsers/chrome.coffee#L16

**Comment by dwelle** [comment]
@jennifer-shehane that fix only seems to work for stable chrome (v66) and not for canary (v68). Can anyone confirm that?

**Comment by brian-mann** [comment]
@dwelle This is a bug in Chrome/Canary - and will likely get fixed in time. We've seen even more bizarre things like this in Canary in the past - and they naturally gets resolved. 

Unfortunately, it's not feasible for us to investigate issues that are chrome bugs ahead of the official releases.

**Comment by jennifer-shehane** [comment]
@dwelle Yes, can confirm - the fix does not work in Canary (v68) 😞

**Comment by dwelle** [comment]
@brian-mann @jennifer-shehane yep, no prob. Was just confirming it's not just me.

**Comment by Igcorreia** [comment]
+1

**Comment by samyakshah** [comment]
@jennifer-shehane how can I supply this --disable-blink-features=RootLayerScrolling ?

**Comment by dwelle** [comment]
@samyakshah Put that code in your `cypress/plugins/index.js` file.

```
module.exports = (on) => {
  on('before:browser:launch', (browser = {}, args) => {
    if (browser.name === 'chrome') {
      args.push('--disable-blink-features=RootLayerScrolling')
      return args
    }
  })
}
```

**Comment by samyakshah** [comment]
Thanks, @dwelle! That worked!

**Comment by EirikBirkeland** [comment]
This was driving me up the wall! Glad to see that this was not a problem with our app!!

**Comment by Nuruddinjr** [comment]
When I run this code on stable chrome 66, screen still shakes.
Fixed with above solutions, but works with screen shaking out of the box.

**Comment by jennifer-shehane** [comment]
We added the Chrome flag `--disable-blink-features=RootLayerScrolling` to Cypress [Version 3.0.0](--disable-blink-features=RootLayerScrolling), so the workaround code should not be necessary anymore - yet, I know the flag did not work for everyone, so will keep issue open.

**Comment by samyakshah** [comment]
Hi Jennifer,

currently I have following code in my index.js, do you recommend removing
it?

on('before:browser:launch', (browser = {}, args) => {
    if (browser.name === 'chrome') {
        args = args.filter((arg) => {
            return arg !== '--disable-blink-features=RootLayerScrolling';
        });
        return args;
    }
});


Thanks,
Samyak

**Comment by jennifer-shehane** [comment]
@samyakshah If you are on Cypress version 3.0.3 or using Chrome 68+ while testing - yes, we recommend removing that code form your `index.js`

**Comment by samyakshah** [comment]
Positive for both of those items. I will remove the block of code.

**Comment by dibya-spak** [comment]
@jennifer-shehane I am facing the same issue on chrome version 101. I have added the above code to my index.js file but it doesn't work. please help me out with this issue!
